File: Palindrome String.cpp
int isPalindrome(string S)
	{

	    int s = 0;
	    int end =   S.length()-1;
	    
	    while(s < end){
	        if(S[s] != S[end]){
	            return 0;
	        }
	       s++;
	       end--;
	    }
	    return 1;
	}

File: class Solution {     int isPalindrome(String S) {         /Palindrome String.java
class Solution {
    int isPalindrome(String S) {
        // code here
        int s =0, e = S.length() -1;
        while(s<e){
            if(S.charAt(s)!=S.charAt(e))
            return 0;
            s++;
            e--;
        }
        return 1;
    }
};
